cluster/images/etcd/rollback/etcd237/pkg/fileutil/fileutil.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2015 CoreOS, Inc.
```

Comment:
@mml @wojtek-t 

Can any of you verify the current solution can work and build?

If everyone is happy about moving the modified dep to third_part/forked/, I will do this after we feel good about this solution.